src/module/skb/bpf.rs

```diff
@@ -359,3 +360,45 @@ pub(super) fn unmarshal_data_ref(raw_section: &BpfRawSection) -> Result<SkbDataR
         dataref: raw.dataref,
     })
 }
+
+/* Keep in sync with "enum ip_conntrack_info" values
+ * (include/uapi/linux/netfilter/nf_conntrack_common.h)
+ */
+const CT_ESTABLISHED: u8 = 0;
+const CT_RELATED: u8 = 1;
+const CT_NEW: u8 = 2;
+const CT_REPLY: u8 = 3;
+const CT_ESTABLISHED_REPLY: u8 = 4;
+const CT_RELATED_REPLY: u8 = 5;
+const CT_UNTRACKED: u8 = 7;
```

Comment:
Right, I thought about it but then considered they should change less often than drop reasons.... or maybe I was just lazy xD

Reply:
> Yes, it's UAPI[1] so it's probably fine to hardcode those values. Runtime inspection still allows for new values to be detected and better handled (printing their in-kernel name).
>

Agree, inspection is more robust for new members however, I think it would be nice to have well defined ct states so that post-processing tools can match on them by enum value rather than string. So, if we build the enum dynamically, what should we do with the new,  unsupported values? Have a `Other(String)` variant?

If new states are added we might also have to change the way we calculate the direction of the flow...